### ISSUE TITLE
Presence Checker - Interface for checking validation of data

### ISSUE THREAD

**Issue by CharlotteHinton** [ServiceInnovationLab/PresenceChecker#9]
As a Citizenship Officer
I can view information about a customer's travel dates over the last five years, and see whether I am eligible to apply  now, and in the near future so that I am able to advise a customer about their eligibility

A / C
* Getting this working for single illustrative customer record is sufficient (bear minimum to allow us to present to TAM Design Authority)
* Should be able to see a count of days in the rolling year when they were present in New Zealand.
* Should be able to see back five years
* Does NOT include status calculation (coming later)

depends on #155
depends on #156
depends on #149

**Comment by Br3nda** [comment]
I am removing "A script for generating fake data (Factory Bot)" - that's in another story

**Comment by Br3nda** [comment]
It would be really useful to have the details of the design to implemented recorded here on the story.

**Comment by joris-rotteveel** [comment]
![localhost_3000_clients_1](https://user-images.githubusercontent.com/13443917/49703468-581f9d80-fc6a-11e8-885f-7c28e044fa02.png)

**Comment by robholmesnz** [comment]
dropped dependency on #107. This story can be completed with single a single illustrative customer record (see A/C).
